draft-yasskin-http-origin-signed-responses.md

```diff
@@ -764,6 +875,50 @@ signed exchange.
 
 Specification: This document
 
+## Internet Media Type application/http-exchange+cbor
+
+Type name:  application
+
+Subtype name:  http-exchange+cbor
+
+Required parameters:  N/A
+
+Optional parameters:  N/A
+
+Encoding considerations:  binary
+
+Security considerations:  see {{security-application-http-exchange}}
+
+Interoperability considerations:  N/A
+
+Published specification:  This specification (see
+{{application-http-exchange}}).
+
+Applications that use this media type:  N/A
+
+Fragment identifier considerations:  N/A
+
+Additional information:
+
+  Deprecated alias names for this type:  N/A
+
+  Magic number(s):  8? 64 68 74 78 67
+
+  File extension(s):  N/A
```

Comment:
Would it be possible to standardize one?

Reply:
Reg: impl side concerns I think we're also fine with leaving it as N/A

Reply:
I'll grab `htxg` which matches the magic number that I pulled out of my hat, but I'm going to invite the HTTPWG to propose other names for the mime type, which could easily change the extension.